protected/humhub/tests/codeception/fixtures/data/setting.php

```diff
@@ -17,6 +17,7 @@
     array('name' => 'maxFileSize', 'value' => '1048576', 'module_id' => 'file'),
     array('name' => 'forbiddenExtensions', 'value' => 'exe', 'module_id' => 'file'),
     array('name' => 'cache.class', 'value' => 'CFileCache', 'module_id' => 'base'),
+    array('name' => 'cache.useApcu', 'value' => '0', 'module_id' => 'base'),
```

Comment:
Shouldn't this also work? Just trying to understand why `0` would be the better choice.
`array('name' => 'cache.useApcu', 'value' => 'CFileCache', 'module_id' => 'base'),`
Or this should also work being Yii2.
`array('name' => 'cache.useApcu', 'value' => 'FileCache', 'module_id' => 'base'),`